utils/deps_verilog.py

```diff
@@ -25,7 +25,27 @@
 
 
 def read_dependencies(inputfile):
-    matches = v_include.findall(inputfile.read())
+    """Read the dependencies out of a verilog file.
+
+    >>> list(read_dependencies(StringIO('''
+    ... `include "a.h"
+    ... `include "b.h" /* Cmt
+    ... `include "c.h"
+    ...    */
+    ... `include "d.h" // Cmt
+    ... // `include "e.h"
+    ...   `include "f.h"
+    ... ''')))
+    ['a.h', 'b.h', 'd.h', 'f.h']
```

Comment:
Probably worth having some paths in here?

Reply:
As example, right?

Reply:
Yeah.

Reply:
Actually, I think that the comments that are already present are self-explanatory enough